### ISSUE TITLE
Error in conversation with Macello in Home

### ISSUE THREAD

**Issue by anubhavpulkit** [systers/PowerUp#345]
## Description
The user completes the Home scenario, then the outro sequence occurs. When the outro completes, the Home scenario screen shows briefly and then the Mine Sweeper games appear. This brief appearance of the Home scenario screen should not occur at all.

## Mocks

[powerup.mov.zip](https://github.com/anitab-org/powerup-iOS/files/4238218/powerup.mov.zip)

### Update [Required]
- [x] [Remove the conversation screen which comes just before Mine Sweeper Game]
or
- [x] [Increase the time suspended time of conversation screen with relevant conversation]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[4] hours

**Comment by anubhavpulkit** [comment]
@sunjunkie please have a look.
Can I work on this issue?

**Comment by sunjunkie** [comment]
Yes. But let's add more detail to the issue description. The user completes the Home scenario, then the outro sequence occurs. When the outro completes, the Home scenario screen shows briefly and then the Mine Sweeper games appears. This brief appearance if the Home scenario screen should not occur at all.

**Comment by anubhavpulkit** [comment]
@sunjunkie is it right now?

**Comment by sunjunkie** [comment]
Remove the 1st paragraph.

**Comment by anubhavpulkit** [comment]
Thank you @sunjunkie

**Comment by anubhavpulkit** [comment]
@sunjunkie I need help on this issue?

**Comment by sunjunkie** [comment]
Please explain your problem.

**Comment by anubhavpulkit** [comment]
Sorry, @sunjunkie I saw your reply but I want to try more times so I can't give reply to this issue but after some more efforts, I can't understand the relation sequence between popup Event and Story sequence.

**Comment by sunjunkie** [comment]
These links provide some background- 
Terminology: https://github.com/anitab-org/powerup-iOS/wiki/Terminology-for-OOC-event-classes-and-utility-web-apps
Story Designer readme:
https://github.com/anitab-org/powerup-story-designer
Cady's notes from GSoC 2018: https://github.com/anitab-org/powerup-iOS/wiki/GSoC-2018-Cadence-Holmes

**Comment by anubhavpulkit** [comment]
Ok, thank you @sunjunkie I read first one last day. I hope I get some help from 2 or 3 links.

**Comment by justkd** [comment]
Hi there. So just to be clear the PopupEvents and StorySequences are _unrelated_. They are separate things entirely. PopupEvents are essentially just notification popups and can be triggered anywhere. StorySequences are the semi-automated dialogue sequences that can occur before and after a scene.

StorySequences aren't pushed with their own view controller. They are just container in a UIView overlaying the current view controller. So, when it's finished, it is disposed of by the scene VC before the minigame VC takes over. The StorySequence class handles removing and disposing of it's own view when the sequence is finished. Resolving this issue might be as simple as not letting it do that...
